react/tic-tac-toe/src/redux/tictactoe/actions.js

```diff
@@ -0,0 +1,16 @@
+export const actions = {
+  GET_HISTORIC: '@@TICTACTOE/GET_HISTORIC'
+};
+
+const actionsCreators = {
+  getHistoric: data => ({
+    type: actions.GET_HISTORIC,
+    payload: data
+  }),
+  requestError: error => ({
```

Comment:
you aren't using this, neither in the reducer nor in the component. Also, this should be specific of each request, because you might do something different in each request error (once you have more than one request). So:

```
getHistoricFailure: error => ({
  type: actions.GET_HISTORIC_ERROR,
  payload: error
})
```

Reply:
gonna ask smth about this tomorrow 
thanks!